### DISCUSSION TITLE
Disable linter stuff

### DISCUSSION THREAD

**Discussion by nothinghalosix** [ixkaito/bathe#31] [General]
Hi, i will include some styles from my theme and get some linter errors. ID, nesting etc.

so how can I disable the linter stuff?

thanks

**Comment by stefancroft** [comment]
This is a little late, but it should be in .eslintrc.js

**Comment by nothinghalosix** [comment]
I wrote this condition in the .stylelintrc.json and it works

"selector-max-id": 5
